skills/csharp/experimental/itsmskill/Dialogs/Teams/CreateTicketTaskModuleView/RenderCreateIncidentHelper.cs

```diff
@@ -0,0 +1,110 @@
+﻿namespace ITSMSkill.Dialogs.Teams.CreateTicketTaskModuleView
+{
+    using System.Collections.Generic;
+    using AdaptiveCards;
+    using ITSMSkill.Dialogs.Teams;
+    using ITSMSkill.Extensions.Teams.TaskModule;
+    using Microsoft.Bot.Schema;
+
+    public class RenderCreateIncidentHelper
+    {
+        public static TaskEnvelope GetUserInput()
+        {
+            var response = new TaskEnvelope
+            {
+                Task = new TaskProperty()
+                {
+                    Type = "continue",
+                    TaskInfo = new TaskInfo()
+                    {
+                        Title = "ImpactTracker",
+                        Height = "medium",
+                        Width = 500,
+                        Card = new Attachment
+                        {
+                            ContentType = AdaptiveCard.ContentType,
+                            Content = TicketDialogHelper.CreateIncidentAdaptiveCard()
+                        }
+                    }
+                }
+            };
+
+            return response;
+        }
+
+        public static TaskEnvelope IncidentAddFailed()
+        {
+            var response = new TaskEnvelope
+            {
+                Task = new TaskProperty()
+                {
+                    Type = "continue",
+                    TaskInfo = new TaskInfo()
+                    {
+                        Title = "Incident Create Failed",
+                        Height = "medium",
+                        Width = 500,
+                        Card = new Attachment
+                        {
+                            ContentType = AdaptiveCard.ContentType,
+                            Content = ImpactTrackerResponseCard("Incident Create Failed")
+                        }
+                    }
+                }
+            };
+
+            return response;
+        }
+
+        public static TaskEnvelope ImpactAddEnvelope()
+        {
+            var response = new TaskEnvelope
+            {
+                Task = new TaskProperty()
+                {
+                    Type = "continue",
+                    TaskInfo = new TaskInfo()
+                    {
+                        Title = "Incident Added",
+                        Height = "medium",
+                        Width = 500,
+                        Card = new Attachment
+                        {
+                            ContentType = AdaptiveCard.ContentType,
+                            Content = ImpactTrackerResponseCard("Incident has been created")
+                        }
+                    }
+                }
+            };
+
+            return response;
+        }
+
+        /// <returns>Adaptive Card.</returns>
```

Comment:
i think if we're adding comments we should follow the default format. if not needed then we shouldn't add comments at all